app/assets/stylesheets/govuk_publishing_components/components/_layout-header.scss

```diff
@@ -28,6 +28,63 @@
   }
 }
 
+.gem-c-layout-header--search-left {
+
+  .gem-c-header__menu-button {
+    position: absolute;
```

Comment:
Looking at the CSS hierarchy it looks like absolute positioning is already defined for this element.

<img width="347" alt="Screenshot 2020-02-06 at 13 29 23" src="https://user-images.githubusercontent.com/861310/73941206-bb47a700-48e4-11ea-9df9-a1d23309eae7.png">